### PR TITLE
Ignore empty laser scan message.

### PR DESCRIPTION
**Pull request by gaschler**
FIXES=#766,#764

### CODE REVIEW

File: cartographer_ros/cartographer_ros/sensor_bridge.cc [hunk 1]
```diff
@@ -181,6 +181,9 @@ void SensorBridge::HandleLaserScan(
     const std::string& sensor_id, const carto::common::Time time,
     const std::string& frame_id,
     const carto::sensor::PointCloudWithIntensities& points) {
+  if (points.points.empty()) {
+    return;
```
Comment (pifon2a) [inline cartographer_ros/cartographer_ros/sensor_bridge.cc]: super-nit: put it in one line, not 3.

Comment (gaschler) [reply cartographer_ros/cartographer_ros/sensor_bridge.cc]: Done

**Comment by gaschler** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user gaschler. Merge queue now has a length of 1.

File: cartographer_ros/cartographer_ros/sensor_bridge.cc (hunk 1 above)
Comment (ojura) [reply cartographer_ros/cartographer_ros/sensor_bridge.cc]: Doesn't the Google coding style mandate always putting braces, even for single-command blocks?

**Comment by wally-the-cartographer** [comment]
Refusing to merge, since the combined checks state is failure

**Pull request merged**